### PR TITLE
implemented new contact form, add some css for transition

### PR DESCRIPTION
**Pull request by locedu**
review this cool website!

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -20,25 +20,22 @@ <h1>SAPIEN</h1>
...
-          
-          <div>
-            <img src="img/quest.png" alt="Movie: Quest For Fire" />
-          </div>
-        
+        <h2>Mission Statement</h2>
+        <p>
+          Lorem ipsum dolor, sit amet consectetur adipisicing elit. Cum
+          molestias assumenda amet omnis ex asperiores ipsam nam. Impedit,
+          veniam necessitatibus? Natus possimus totam earum cupiditate maiores
```
Comment (CLSnapp) [inline index.html:31]: May need to backspace this area, to make it look cleaner.

File: style.css [hunk 2]
```diff
@@ -126,14 +126,49 @@ section {
...
 img {
-  /* max-width: 100%; */
   height: auto;
-  max-width: 300px;
+  max-width: 25vh;
   width: -moz-available;
   width: -webkit-fill-available;
 }
 
+.image-fadeIn {
```
Comment (CLSnapp) [inline style.css:137]: nicely done with the fade-in

**Review by CLSnapp — APPROVED** [review]
LGTM! I did suggest a backspace on one part, to make the code look cleaner.

**Comment by locedu** [comment]
thanks for the review!

**Pull request merged**